domain/model/state/state.go

```diff
@@ -307,6 +307,39 @@ SELECT type FROM model_type;
 	})
 }
 
+// List returns a list of all model UUIDs.
+// The list of models returned are the ones that are just present in the model
+// manager list. This means that the model is not deleted.
```

Comment:
Doesn't the fact that a model uuid in the list of returned model uuids mean that it's not deleted by definition? Since there's no "deleted" column, when would we have a model db that is considered "deleted" and the uuid not in this list, necessitating the need for the comment clarification? When we delete a model, wouldn't we mark the uuid in this table as "deleting" or something so that a suitable worker can be triggered to nuke the model db and then remove the row when done?

Reply:
Updated the wording.